test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -1462,15 +1462,15 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 				}
 			}
 
-			for _, azureFileStorageClass := range []string{"azurefile"} {
-				sc, err := storageclass.Get(azureFileStorageClass)
-				Expect(err).NotTo(HaveOccurred())
-				Expect(sc.Provisioner).To(Equal(azureFileProvisioner))
-				Expect(sc.VolumeBindingMode).To(Equal("Immediate"))
-				if isUsingAzureFileCSIDriver && common.IsKubernetesVersionGe(eng.ExpandedDefinition.Properties.OrchestratorProfile.OrchestratorVersion, "1.16.0") {
-					Expect(sc.AllowVolumeExpansion).To(BeTrue())
-				}
-			}
+			// for _, azureFileStorageClass := range []string{"azurefile"} {
```

Comment:
Instead of commenting out, could we instead wrap this block and hard-code the provioner?

```
if isUsingAzureFileCSIDriver, _ = eng.HasAddon(common.AzureFileCSIDriverAddonName); isUsingAzureFileCSIDriver {
  azureFileProvisioner = "file.csi.azure.com"
  ...
```

Reply:
done